fairchive-webapp/src/main/java/edu/harvard/iq/dataverse/validation/field/DatasetFieldValidationDispatcher.java

```diff
@@ -77,18 +82,34 @@ private FieldValidationResult validateField(DatasetField field) {
     private boolean isRequiredInDataverse(DatasetField field) {
         DatasetFieldType fieldType = field.getDatasetFieldType();
         if (fieldType.isRequired()) {
-            return true;
+            return isParentForConditionalRenderingSelected(field);
         }
 
         Dataverse dataverse = field.getTopParentDatasetField()
                 .getDatasetVersion()
                 .getDataset()
                 .getOwner().getMetadataBlockRootDataverse();
-        return dataverse.getDataverseFieldTypeInputLevels().stream()
+        boolean inputLevelRequired =  dataverse.getDataverseFieldTypeInputLevels().stream()
                 .filter(inputLevel -> inputLevel.getDatasetFieldType().equals(field.getDatasetFieldType()))
                 .map(DataverseFieldTypeInputLevel::isRequired)
                 .findFirst()
                 .orElse(false);
+
+        return inputLevelRequired && isParentForConditionalRenderingSelected(field);
+    }
+
+    private boolean isParentForConditionalRenderingSelected(DatasetField field) {
```

Comment:
I'm not too keen on the method name. From the method name I would tell that we check whether something is selected on the parent field.
Maybe something like `isFieldRendered(...)` would be more precise? Since we not only check whether condition for conditional rendering is true, but we also return true when no conditional rendering is defined.